fetch.sh

```diff
@@ -200,7 +220,9 @@ cpuspeed () {
 
 # Memory (Configurable with "-M" and "--memory" at launch)
 # Print the total amount of ram and amount of ram in use
-memory=$(free -m | awk '/Mem:/ {printf $3 "MB / " $2 "MB"}')
+usedmem=$(free -m | awk '/cache:/ {print $3}')
```

Comment:
This calls free and awk twice, I don't think the readability benefit is worth it.